community/mpdas/depends

```diff
@@ -0,0 +1,2 @@
+curl
+libmpdclient
```

Comment:
Where does this dependency come from? Is it in Community?

Reply:
https://github.com/kisslinux/community/tree/master/community/libmpdclient